### PR TITLE
Update: Pre-test for OpenMS 2.7

### PR DESCRIPTION
**Pull request by jpfeuffer**
Will update OpenMS to the newest version if successful.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by bgruening** [comment]
> 16:00:29 BIOCONDA INFO (OUT) Failed to download or patch source. Please see build log for info.

I see that here.

File: recipes/openms-meta/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+python:
+  - 3.9.*  *_cpython
+  - 3.8.*  *_cpython
+python_impl:
+  - cpython
```
Comment (bgruening) [inline recipes/openms-meta/conda_build_config.yaml:5]:
```suggestion
  - cpython
c_compiler_version:            # [linux]
  - 9                                      # [linux]
cxx_compiler_version:         # [linux]
  - 9                                      # [linux]
```

You could also try version 11.

**Comment by jpfeuffer** [comment]
https://support.circleci.com/hc/en-us/articles/4410707277083-Context-deadline-exceeded-after-1-hour-Build-timed-out-Free-tier-only-

Bad news. 1h is going to be close to impossible even with the biggest machine, considering we would like to build at least two or even better 3 python versions.

**Comment by bgruening** [comment]
@jpfeuffer GCC 11 fails to compile.

**Comment by jpfeuffer** [comment]
Ah I see. I will provide a patch. Should hopefully only be rearranging includes.
Merry Xmas btw 🌲

File: recipes/openms-meta/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
 
 build:
   number: 1
-
-# Try this when building with GUI
-#extra:
-#  container:
-#    extended-base: True
+  # Problems with memory usage on gcc+circleci. Needs to be built manually and pushed. And to debug macos
+  skip: True  # [linux]
```
Comment (bgruening) [inline recipes/openms-meta/meta.yaml:19]: @jpfeuffer we are not using CircelCI anymore. Do you dare to test this again here?

Comment (jpfeuffer) [reply recipes/openms-meta/meta.yaml:19]: Yep it worked. Just needed to fix macOS now and did not want to wait so long all the time ;)

**Comment by jpfeuffer** [comment]
@BiocondaBot please add label

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@jpfeuffer :1st_place_medal: :tada: 

Thanks for fighting with this so long, very much appreciated!

**Pull request merged**